### ISSUE TITLE
Elevator Caller broken

### ISSUE THREAD

**Issue by Omegahedron** [MineMaarten/PneumaticCraft#894]
On my server I have a 3x3 elevator with 4 callers, which was working fine until I applied a redstone signal to the bottom caller. When I did that every caller but the bottom one turned blank, and the bottom one now only displays one floor (itself). I tried replacing all of the callers, replacing the elevator bases, moving the callers to different sides of the elevator - nothing works. Any help with this issue would be appreciated.

**Comment by bookerthegeek** [comment]
Did you remove the redstone signal from the bottom caller?

**Comment by Omegahedron** [comment]
Yeah, it was just a pulse from a button. So apparently I was able to get the callers to display the floors correctly again by removing them, breaking and replacing part of the elevator frame, and then replacing them. So far they're responding normally to redstone signals (hopefully I don't jinx myself), but I still don't know what caused them to become bugged in the first place. I forgot to mention I'm playing on FTB Inventions (1.7.10).
